syntax = "proto3";

package greet;

message GreetRequest {
    string name = 1;
}

message GreetResponse {
    string message = 1;
}

service Greeter {
    rpc SayHello (GreetRequest) returns (GreetResponse);
}